packages/cli/src/UserManagement/UserManagementHelper.ts

```diff
@@ -2,27 +2,34 @@
 /* eslint-disable @typescript-eslint/no-non-null-assertion */
 /* eslint-disable import/no-cycle */
 import { IsNull, Not } from 'typeorm';
-import { Db, ResponseHelper } from '..';
+import { Db, GenericHelpers, ResponseHelper } from '..';
 import config = require('../../config');
-import { User } from '../databases/entities/User';
+import { MAX_PASSWORD_LENGTH, MIN_PASSWORD_LENGTH, User } from '../databases/entities/User';
 import { PublicUser } from './Interfaces';
 
-export function isEmailSetup(): boolean {
-	const emailMode = config.get('userManagement.emails.mode') as string;
-	return !!emailMode;
+export const isEmailSetUp = Boolean(config.get('userManagement.emails.mode'));
+
+export function getInstanceDomain(): string {
+	let domain = GenericHelpers.getBaseUrl();
+	if (domain.endsWith('/')) {
+		domain = domain.slice(0, domain.length - 1);
+	}
+
+	return domain;
 }
 
 export async function isInstanceOwnerSetup(): Promise<boolean> {
 	const users = await Db.collections.User!.find({ email: Not(IsNull()) });
 	return users.length !== 0;
 }
 
+// TODO: Enforce at model level
```

Comment:
Hashing changes length so pass cannot be validated at model level unless we hash there as well - probably worth discussing. Since the helper is used by multiple namespaces, I decided to leave it untouched.